wp-admin/js/customize-controls-site-icon-maskable.js

```diff
@@ -28,8 +29,59 @@ wp.customize(
 				// Set initial active state.
 				updateActive();
 
+				/**
+				 * Validate site icons for its presence and size.
+				 */
+				const iconValidation = () => {
+					const iconData = wp.customize
+						.control('site_icon')
+						.container.find('img.app-icon-preview');
+					const NotificationData = {
+						dismissible: true,
```

Comment:
There won't be a need to be `dismissible` if the notification automatically is removed when the issue is fixed.